resources/99-qtile.rules

```diff
@@ -0,0 +1,26 @@
+# for controlling LCD backlight
+ACTION=="add", SUBSYSTEM=="backlight", RUN+="qtile udev backlight --device %k"
+
+# keyboard backlight
+ACTION=="add", SUBSYSTEM=="leds",      RUN+="qtile udev backlight --device %k"
```

Comment:
```suggestion
ACTION=="add", SUBSYSTEM=="backlight", RUN+="qtile udev backlight --device %k  --group video"

# keyboard backlight
ACTION=="add", SUBSYSTEM=="leds",      RUN+="qtile udev backlight --device %k --group video"
```
Just an idea...

Reply:
IMO, we should use the same group (sudo, video, whatever) for everything. I picked sudo because I assumed that most single user systems have the user qtile runs as in the sudo group. We could add a new `qtile` group, but that would introduce even more setup than already exists for people installing by source, so I didn't opt for that.

Reply:
I'm fine with using sudo group as long as it's clear to users. Agreed, a single group would be preferable.